### PR TITLE
Wizard activity material update fix

### PR DESCRIPTION
**Pull request by marcelinkaaa**
@aldenml @gubatron 
This was more straight forward. I added a new style to the activity, made adjustment so that we are using material buttons, and extended AbstractActivity2. 
Grzesiek added a quick fix for selecting a storage. 

screenshot_20161216-115648
screenshot_20161216-115652
screenshot_20161216-115656

### CODE REVIEW

File: android/res/values/styles.xml [hunk 1]
```diff
@@ -581,31 +581,66 @@
...
-        <item name="android:textSize">@dimen/text_size_small</item>
-        <item name="android:fontFamily">sans-serif-light</item>
+        <item name="android:paddingLeft">15dp</item>
+        <item name="android:paddingRight">15dp</item>
+        <item name="android:textColor">@android:color/white</item>
+        <item name="android:textSize">@dimen/body_text</item>
+    </style>
+
+    <style name="WizardText.Header">
+        <item name="android:fontFamily">sans-serif-medium</item>
```
Comment (aldenml) [inline android/res/values/styles.xml]: Can you verify that this font do exists in API 16?

File: android/src/com/frostwire/android/StoragePicker.java [hunk 2]
```diff
@@ -40,7 +40,7 @@
 
     public static final String ACTION_OPEN_DOCUMENT_TREE = "android.intent.action.OPEN_DOCUMENT_TREE";
 
-    public static final int SELECT_FOLDER_REQUEST_CODE = 1267123;
+    public static final int SELECT_FOLDER_REQUEST_CODE = 126;
```
Comment (aldenml) [inline android/src/com/frostwire/android/StoragePicker.java]: why the change here?

File: android/src/com/frostwire/android/gui/activities/WizardActivity.java [hunk 3]
```diff
@@ -31,6 +31,7 @@
 import com.frostwire.android.core.Constants;
 import com.frostwire.android.gui.util.UIUtils;
 import com.frostwire.android.gui.views.AbstractActivity;
```
Comment (aldenml) [inline android/src/com/frostwire/android/gui/activities/WizardActivity.java]: import cleanup?

File: android/src/com/frostwire/android/StoragePicker.java (hunk 2 above)
Comment (marcelinkaaa) [inline android/src/com/frostwire/android/StoragePicker.java]: from grzesiek "seems like it is an internal android thing that when we switched to compat version the requestCode for starting activity has to be a 16bit unsigned integer"

**Comment by aldenml** [comment]
it looks clean to me, take care of the font comment and then I can merge

**Comment by marcelinkaaa** [comment]
Doing it now :)

**Comment by marcelinkaaa** [comment]
fix added
